packages/babel-cli/src/babel/file.js

```diff
@@ -103,19 +103,21 @@ export default function(commander, filenames, opts) {
     });
 
     process.stdin.on("end", function() {
-      results.push(
-        util.transform(
-          commander.filename,
-          code,
-          defaults(
-            {
-              sourceFileName: "stdin",
-            },
-            opts,
-          ),
+      util.transform(
+        commander.filename,
+        code,
+        defaults(
+          {
+            sourceFileName: "stdin",
+          },
+          opts,
         ),
+        function(err, result) {
+          if (err) console.error(err.stack);
```

Comment:
To be most consistent with the old behavior, this should probably be `throw err;`